tests/Core/Ast/Parser/AnnotationReferenceExtractorTest.php

```diff
@@ -35,7 +34,7 @@ public function testPropertyDependencyResolving(): void
         $annotationDependency = $astClassReferences[0]->dependencies;
 
         self::assertCount(2, $astClassReferences);
-        self::assertCount(9, $annotationDependency);
```

Comment:
[ParserFactory](https://github.com/nikic/PHP-Parser/blob/master/UPGRADE-5.0.md#changes-to-comment-assignment) Might be the reason for the changed behavior.
Needs more recherche!

Reply:
Sorry for takins so long. Yes you are right about the cause, I took some time digging through php-parser and PHPStan commits (when PHP upgraded) to find and implement the fix. It is in https://github.com/qossmic/deptrac-src/pull/103. Feel free to reuse the code from there for this PR or merge mine, makes no difference to me.